llvmlite/tests/test_binding.py

```diff
@@ -692,6 +692,71 @@ def jit(self, mod, target_machine=None):
         return llvm.create_mcjit_compiler(mod, target_machine)
 
 
+class TestOrcMCJit(BaseTest, JITWithTMTestMixin):
+    """
+    Test JIT engines created with create_orc_mcjit_compiler().
+    """
+
+    def jit(self, mod, target_machine=None):
+        if target_machine is None:
+            target_machine = self.target_machine()
+        return llvm.create_orc_mcjit_compiler(mod, target_machine)
+
+    def test_object_cache_getbuffer(self):   # overrides
+        # The ORC JIT needs a different test because it will fail at the
+        # second phase if loading function "sum" in a module without one.
+        # Instead, we will supply another "sum" implementation and make sure it
```

Comment:
Is the behavior in the MCJIT-version documented? 

It seems OrcJIT is stricter here.  Due to its laziness, Orc first tries to find a IR module that has the symbol.  Once found, it checks if it is already compiled and do its thing.  If not found, it stops and return NULL.